### PR TITLE
DDF-2236:updated docs to 2.9.2

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

updates http://codice.org/ddf to include the latest version of the documentation.

#### Who is reviewing it?

@pklinef @shaundmorris 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@pklinef
@shaundmorris

#### How should this be tested?

Verify links work from Documentation-versions.html and lead to the correct versions.

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: documentation.html [hunk 1]
```diff
@@ -31901,16 +31940,16 @@ <h4 id="_creating_a_kar_file"><a class="anchor" href="#_creating_a_kar_file"></a
                     same feature twice.
                     Refer to Karaf forum posting at http://karaf.922171.n3.nabble.com/Duplicate-feature-repository-entry-using-archive-kar-to-build-deployable-applications-td3650850.html
                     --&gt;</span>
-                    <span class="tag">&lt;resourcesDir&gt;</span>/Users/phillip/Documents/workspace/release/ddf/distribution/docs/target/doesNotExist<span class="tag">&lt;/resourcesDir&gt;</span>
+                    <span class="tag">&lt;resourcesDir&gt;</span>/Users/smorris/ddf/ddf/distribution/docs/target/doesNotExist<span class="tag">&lt;/resourcesDir&gt;</span>
```
Comment (pklinef) [inline documentation.html]: We need to fix this so the documentation does not expose the directory used by the person doing the release.

**Comment by pklinef** [comment]
@ricklarsen It looks like you are just adding the generated documentation, is that correct?  Is the quick start page generated as well?

**Comment by pklinef** [comment]
👍

File: documentation.html (hunk 1 above)
Comment (ricklarsen) [inline documentation.html]: agreed.

**Comment by ricklarsen** [comment]
Yes. the quickstart page is also generated.

File: documentation.html (hunk 1 above)
Comment (ricklarsen) [inline documentation.html]: Found what was causing this. It's fixed in DDF-2396. https://github.com/codice/ddf/pull/1206

File: documentation.html [hunk 2]
```diff
@@ -31901,16 +31940,15 @@ <h4 id="_creating_a_kar_file"><a class="anchor" href="#_creating_a_kar_file"></a
...
-                    <span class="tag">&lt;resourcesDir&gt;</span>/Users/phillip/Documents/workspace/release/ddf/distribution/docs/target/doesNotExist<span class="tag">&lt;/resourcesDir&gt;</span>
-
+                    
                     <span class="comment">&lt;!--
-                    Location of the features.xml file. If it references properties that need to be filtered, e.g., 2.9.1, it will need to be
+                    Location of the features.xml file. If it references properties that need to be filtered, e.g., 2.9.2, it will need to be
                     filtered by the maven-resources-plugin.
                     --&gt;</span>
-                    <span class="tag">&lt;featuresFile&gt;</span>/Users/phillip/Documents/workspace/release/ddf/distribution/docs/target/classes/features.xml<span class="tag">&lt;/featuresFile&gt;</span>
+                    <span class="tag">&lt;featuresFile&gt;</span>/Users/smorris/ddf/ddf/distribution/docs/target/classes/features.xml<span class="tag">&lt;/featuresFile&gt;</span>
```
Comment (shaundmorris) [inline documentation.html]: How is this path getting generated?

File: extending.html [hunk 3]
```diff
@@ -2224,16 +2224,15 @@ <h4 id="_creating_a_kar_file"><a class="anchor" href="#_creating_a_kar_file"></a
...
                     --&gt;</span>
-                    <span class="tag">&lt;resourcesDir&gt;</span>/Users/phillip/Documents/workspace/release/ddf/distribution/docs/target/doesNotExist<span class="tag">&lt;/resourcesDir&gt;</span>
 
                     <span class="comment">&lt;!--
-                    Location of the features.xml file. If it references properties that need to be filtered, e.g., 2.9.1, it will need to be
+                    Location of the features.xml file. If it references properties that need to be filtered, e.g., 2.9.2, it will need to be
                     filtered by the maven-resources-plugin.
                     --&gt;</span>
-                    <span class="tag">&lt;featuresFile&gt;</span>/Users/phillip/Documents/workspace/release/ddf/distribution/docs/target/classes/features.xml<span class="tag">&lt;/featuresFile&gt;</span>
+                    <span class="tag">&lt;featuresFile&gt;</span>/Users/smorris/ddf/ddf/distribution/docs/target/classes/features.xml<span class="tag">&lt;/featuresFile&gt;</span>
```
Comment (shaundmorris) [inline extending.html]: Same question on path.

**Comment by shaundmorris** [comment]
👍

File: documentation.html (hunk 2 above)
Comment (ricklarsen) [inline documentation.html]: There's a `{basedir}` in a code sample that asciidoctor apparently has a default variable for. I changed the syntax in one of my PRs to stop it happening, but I'll have to delete it manually from these ones that are already generated.

File: extending.html (hunk 3 above)
Comment (ricklarsen) [inline extending.html]: see above.

**Comment by ricklarsen** [comment]
:shipit: ❗

**Pull request merged**